csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,106 +5,88 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static List<string> Tasks { get; set; }
 
         static void Main(string[] args)
         {
-            TL = new List<string>();
-            int variable = 0;
+            Tasks = new List<string>();
+            int selectedOption = 0;
+
             do
             {
-                variable = ShowMainMenu();
-                if (variable == 1)
+                selectedOption = ShowMainMenuAndGetChoice();
+                if (selectedOption == 1)
                 {
-                    ShowMenuAdd();
+                    AddTask();
                 }
-                else if (variable == 2)
+                else if (selectedOption == 2)
                 {
-                    ShowMenuDos();
+                    RemoveTask();
                 }
-                else if (variable == 3)
+                else if (selectedOption == 3)
                 {
-                    ShowMenuTres();
+                    ShowCurrentTasks();
                 }
-            } while (variable != 4);
+            } while (selectedOption != 4);
         }
-        /// <summary>
-        /// Show the main menu 
-        /// </summary>
-        /// <returns>Returns option indicated by user</returns>
-        public static int ShowMainMenu()
+
+        public static int ShowMainMenuAndGetChoice()
         {
             Console.WriteLine("----------------------------------------");
             Console.WriteLine("Enter the option to perform: ");
             Console.WriteLine("1. New task");
             Console.WriteLine("2. Remove task");
             Console.WriteLine("3. Pending tasks");
             Console.WriteLine("4. Exit");
-
-            // Read line
-            string line = Console.ReadLine();
-            return Convert.ToInt32(line);
+            
+            return Convert.ToInt32(Console.ReadLine());
```

Comment:
In some cases, it is better to have more lines to have a more readable code. I suggest keeping these 2 instructions in 2 lines. Also, would be fine if you validate the correct input according to the menu options you have.
```suggestion
            string menuSelected = Console.ReadLine();
            return Convert.ToInt32(menuSelected);
```